lib/active_admin/views/index_as_table.rb

```diff
@@ -213,7 +213,7 @@ def build(page_presenter, collection)
         table_options = {
           id: "index_table_#{active_admin_config.resource_name.plural}",
           sortable: true,
-          class: "index_table index",
+          class: "index-table js-index-table",
```

Comment:
So not to use a presentation class in JS. Its best to separate the two that way as either one changes you aren't breaking the other. Quite handy. I'd like to do this more throughout the project. Look up commits for answers first as I try my best to write detail commit messages on why I certain changes. For this change the commit is: https://github.com/javierjulio/activeadmin/commit/df4f347305195a088f901093a6e4055fb8221f6f